erpnext/accounts/doctype/pos_closing_entry/test_pos_closing_entry.py

```diff
@@ -26,6 +26,8 @@ def tearDown(self):
 		frappe.db.sql("delete from `tabPOS Profile`")
 
 	def test_pos_closing_entry(self):
+		is_allow_neg = frappe.db.get_single_value('Stock Settings', 'allow_negative_stock')
+		frappe.db.set_value('Stock Settings', 'Stock Settings', 'allow_negative_stock', 1)
```

Comment:
Way too much duplicate code, should be a decorator instead, or it applies class-wide then setupclass/teardownclass.